src/AppBundle/Service/ReportStatusService.php

```diff
@@ -416,7 +416,11 @@ public function getProfDeputyCostsState()
         // If costs are only fixed, SCCO question is not required (DDPB-2506)
         $isSccoValid = $onlyFixedTicked || $this->report->getProfDeputyCostsAmountToScco();
 
-        if ($isRemainingValid && $isInterimValid && $isFixedValid && $isSccoValid) {
+        // Its possible to submit an empty breakdown of other costs, however this will still persist empty entities
+        // but its presence indicates that the deputy has actively submitted these and therefore seen the summary page
+        $hasOtherCostsSubmitted = $this->report->hasProfDeputyOtherCosts();
```

Comment:
I think if you rename the var to `$hasSubmittedOtherCostsForm` (or similar) then you can remove this long comment.

Reply:
Done